doc/building.md

```diff
@@ -45,12 +46,13 @@ directory toolchain. Target toolchain directory further will be referred as `$TO
 
 If using **gcc**: add `$TOOLCHAIN_TARGET_DIR/bin` to `$PATH`.
 
-If using **LLVM**: `export CLANG_PATH=$TOOLCHAIN_TARGET_DIR/bin`
+If using **LLVM**: `export CLANG_PATH=$TOOLCHAIN_TARGET_DIR`.
 
 ## Step 5: Build
 
-Do `make -j <nproc> iso` where nproc is the number of threads you want the build to use.
+Do `make -j <nproc> liveiso` where nproc is the number of threads you want the build to use.
 
-You may want to edit `scripts/iso.sh` to edit grub-mkrescue command name for your distro.
+Note that RHEL, Fedora and other RHEL-derived distributions install grub-mkrescue as
+grub2-mkrescue. Edit `scripts/iso.sh` appropriately.
```

Comment:
I don't think this is the correct fix. If I write the fix in scripts/iso.sh, can you test it?

Meanwhile, delete the RHEL, Fedora, yadda yadda paragraph.